client/components/data/query-billing-transaction/index.jsx

```diff
@@ -25,7 +25,7 @@ function QueryBillingTransaction( { transactionId } ) {
 }
 
 QueryBillingTransaction.propTypes = {
-	transactionId: PropTypes.oneOfType( PropTypes.string, PropTypes.number ).isRequired,
+	transactionId: PropTypes.oneOfType( [ PropTypes.string, PropTypes.number ] ).isRequired,
```

Comment:
This change is unrelated to this PR, but fixes a PropType warning in the console.